farm/modeling/language_model.py

```diff
@@ -1046,11 +1070,10 @@ def save(self, save_dir):
         :param save_dir: The directory in which the model should be saved.
         :type save_dir: str
         """
-        raise NotImplementedError
         #save model
-        # self.model.save(save_dir=save_dir)
-        # #save config
-        # self.save_config(save_dir=save_dir)
+        self.model.save(save_dir=save_dir)
```

Comment:
For me this hangs for normal sized (2-3GB) Glove/Word2Vec embeddings. 
I am unsure why, could you test on your machine?

Reply:
It's not hanging for me, but it's crazily slow (~ 15min for german glove). I added a progress bar now to make it more transparent. We should then very soon accelerate this in a separate PR - potentially using numpy.savetxt or changing the format to h5py / torch .bin ...